covid_graphs/covid_graphs/scripts/server.py

```diff
@@ -71,9 +72,26 @@ def covid19_predictions_redirect():
 def _run_flask_server(server: Flask, data_dir: Path):
     _create_prediction_apps(server=server, data_dir=data_dir)
     _create_simulation_apps(server=server, data_dir=data_dir)
+    _create_rest(server=server, data_dir=data_dir)
     server.run(host="0.0.0.0", port=8081)
 
 
+def _create_rest(data_dir: Path, server: Flask):
+    rest = Rest(data_dir=data_dir)
+
+    @server.route("/covid19/data/<country>")
+    def covid19_country_data(country):
+        return rest.get_country_data(country)
+
+    @server.route("/covid19/predictions/list/")
```

Comment:
Is it common to mix user visible API with internal API?

`/covid19/predictions/all` is user-facing, while `/covid19/predictions/list` is for internal use. I find it odd. Could we add `/rest` prefix somewhere?

Also, I'm worried someone will discover this API and fetch JSON data from us. Is it possible to only bind it to the localhost? Perhaps some nginx magic is needed here.

Reply:
1. `/rest`, sure.
1. hide JSON data - no. It's by design, because UI logic runs completely in browser. BTW: current implementation with dash is already sending graph data using HTTP+JSON. It's not that nice as this api, but it's still possible to fetch that data.

Reply:
Right